dataset/src/test/scala/frameless/TypedDatasetSuite.scala

```diff
@@ -6,11 +6,7 @@ import org.scalactic.anyvals.PosZInt
 import org.scalatest.FunSuite
 import org.scalatest.prop.Checkers
 
```

Comment:
I like the explanation. So with Job TypedDataset is always lazy, whereas vanilla Datasets require some tribal knowledge of which operations are lazy and which are not.